app/src/main/java/cm/aptoide/pt/home/bundles/BundlesResponseMapper.java

```diff
@@ -290,8 +294,11 @@ private ActionItem map(ActionItemResponse viewObject) {
           if (widget.getType()
               .equals(Type.NEWS_ITEM)) {
             return HomeBundle.BundleType.NEWS_ITEM;
-          } else {
+          } else if (widget.getType()
+              .equals(Type.IN_GAME_EVENT)) {
             return HomeBundle.BundleType.IN_GAME_EVENT;
+          } else {
+            return HomeBundle.BundleType.APP_COMING_SOON;
```

Comment:
This might be an error that was already in prod. We should return APP_COMING_SOON only if the widget type is of that type. Return UNKNOWN on else. I'm not sure what is the current behavior (if this version in the future receives new types of PROMO bundles)